packages/localizations/src/en-US.ts

```diff
@@ -616,11 +616,25 @@ export const enUS: LocalizationResource = {
         automaticSuggestionOption__description:
           'Users receive a suggestion to request to join, but must be approved by an admin before they are able to join the organization.',
         formButton__save: 'Save',
+        calloutLabels: {
+          prefix_single: 'There is',
+          prefix_many: 'There are',
+          pendingInvitationCount_single: '1 pending invitation',
+          pendingInvitationCount_many: '{{count}} pending invitations',
+          pendingSuggestionCount_single: '1 pending suggestion',
+          pendingSuggestionCount_many: '{{count}} pending suggestions',
+          suffix: 'sent to users. Changing the enrollment mode will only affect new users.',
```

Comment:
Wouldn't it be simpler if we used 2 sentences without breaking it down into prefixes and suffixes? I understand that you're trying to avoid duplication here, but is it worth it?

Reply:
The thing is that we need multiple combinations:

1. There is 1 pending invitation sent to users ....
2. There is 1 suggestion sent to users ....
3. There are 1 pending invitation and 1 pending suggestion sent to users ....
4. There are 2 pending invitations and 1 pending suggestion sent to users ....
5. There are 1 pending invitation and 2 pending suggestions sent to users ....
6. There are 2 pending invitations and 2 pending suggestions sent to users ....